switchboard/base.py

```diff
@@ -32,12 +32,17 @@ def __init__(self, cache=None, timeout=30):
     def __getitem__(self, key):
         self._populate()
         try:
-            return self._cache[key]
+            value = self._cache[key]
```

Comment:
Unfortunately not:

~~~~
>> foo = None
>>> foo.get('bar', 'baz')
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
AttributeError: 'NoneType' object has no attribute 'get'
~~~~

The alternate approach I considered was checking to see `if self._cache is None`, but I ultimately decided to go with the above since it is an exception condition.